tools/ds-identify

```diff
@@ -1137,8 +1145,73 @@ dscheck_SmartOS() {
     return ${DS_NOT_FOUND}
 }
 
+none_read_always_found_setting() {
+    # the 'always_found' setting for fallback datasource controls
+    # behavior of ds-identify with respect to cloud-init, whether it
+    # should be enabled or not, and wheather ds-identify should
+    # filter-out datasources to leave just the fallback datasource
+    # enabled.
+    # order of precedence is:
+    #  1. builtin setting here cloud-init/ds-identify builtin
+    #  2. ds-identify config
+    #  3. system config (/etc/cloud/cloud.cfg.d/*None*.cfg)
+    #  4. kernel command line (undocumented)
+    #  5. user-data or vendor-data (not available here)
```

Comment:
Why include this when ds-identify does not parse user-data/vendor-data?  Including this value in user-data/vendor-data isn't useful in that cloud-init will have already been disabled or if activated it will specify a detected datasource (which will run instead of DataSourceNone) ?